spacy/cli/templates/quickstart_training.jinja

```diff
@@ -585,6 +597,17 @@ no_output_layer = false
 {%- endif %}
 {% endif %}
 
+{% if "llm" in components -%}
+[components.llm]
+factory = "llm"
+
+[components.llm.model]
+@llm_models = "{{ llm_spec['model'] }}"
+
+[components.llm.task]
+@llm_tasks = "{{ llm_spec['task'] }}"
+{% endif -%}
+
```

Comment:
I think (depending on exactly how this gets designed in the end), it should be possible to only have this block once because it does not depend on the listener setup.